.travis.yml

```diff
@@ -14,10 +14,9 @@ addons:
       - libssl-dev
       - python3-dev
 
-before_script:
-  - pip install coala-bears
-  - pip install -r requirements.txt
-  - pip install -r test-requirements.txt
+before_install:
+  - sed -i '1s/^/coala-bears/' requirements.txt
```

Comment:
this regex creates a malformed dep since it just mushes it together. add a newline after the sub? https://travis-ci.org/Mixih/corobo/builds/319014020

Reply:
oops ;-)